File: web-development/level-1/helsinki-moocs/python/week03/ex55.py
word_1 = input('Type the first word: ')
word_2 = input('Type the seconds word: ')

if word_2 in word_1:
    print('The word \'', word_2, '\' is found in the word \'', word_1, '\'')
else:
    print('The word \'', word_2, '\' is NOT found in the word \'', word_1, '\'')

File: web-development/helsinki-mooc/java/ex55.java
import java.util.Scanner;

public class ex55 {
  public static void main(String[] args) {
    Scanner input = new Scanner(System.in);

    System.out.print("Type first word: ");
    String first_word = input.nextLine();

    System.out.print("Type second word: ");
    String second_word = input.nextLine();

    if (first_word.contains(second_word)) {
      System.out.println(second_word + ", found in the word, " + first_word);
    }
    else {
      System.out.println("Two words are not connected");
    }
  }
}
